actors/builtin/storage_market/storage_market_actor_state.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/filecoin-project/specs-actors/actors/util/adt"
 )
 
-const epochUndefined = abi.ChainEpoch(-1)
+const epochUndefined = abi.ChainEpoch(0)
```

Comment:
Please don't do this here. I see https://github.com/filecoin-project/specs/issues/874 but it needs broader acceptance. The genesis epoch isn't something we get to define here. The genesis may well have deals and this will cause us problems.